.github/workflows/vercel-docs-preview.yaml

```
@@ -0,0 +1,51 @@
+  cancel-in-progress: true
+
+jobs:
+  Deploy-Docs-Preview:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    env:
+      COMMIT_SHA: ${{ github.sha }}
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)